Doc updates for migration (#1263)
newrelic/newrelic-quickstarts: CONTRIBUTING.md

applies to all platforms and venues related to this project; please follow it in all
your interactions with the project and its participants.

## Quickstarts contributor guidelines

> Review our comprehensive [Developer Guide](https://developer.newrelic.com/contribute-to-quickstarts/) to get starting building your quickstart!

We encourage all contributors to actively engage in the creation and maintenance of quickstarts. Whether you work at New Relic or use New Relic as a customer, the community is open to your expertise!

- `Step 1`: Review the [quickstart definitions section](#quickstart-definitions) and the [quickstart Template Config](./_template/quickstarts/example-quickstart/config.yml) for a definition of how to create a quickstart.
- `Step 2`: Review the [documentation](https://github.com/newrelic/newrelic-quickstarts/blob/main/docs) for structure and limits you need to consider.
- `Step 3`: Create your quickstart!
- `Step 4`: Submit a PR!
- `Step 5`: Resolve feedback from code reviews.
- `Step 6`: After approval, merge your PR.

### Quickstart PR review workflow

> When you submit a PR for a new our existing quickstart the follow workflow is executed.

Reviewer: ```suggestion
When you submit a PR for a new or existing quickstart the following workflow is executed.
```

> - `Step 1`: A new PR is added the `To do` column of the [NR1 Community Quickstarts project board](https://github.com/newrelic/newrelic-quickstarts/projects/3)

Reviewer: ```suggestion
- `Step 1`: A new PR is added in the `To do` column of the [NR1 Community Quickstarts project board](https://github.com/newrelic/newrelic-quickstarts/projects/3)
```

> - `Step 2`: A repo maintainer will review the PR and provided feedback. The PR will move to the `In progress` column.

Reviewer: ```suggestion
- `Step 2`: A repo maintainer will review the PR and provide feedback. The PR will move to the `In progress` column.
```

### Pull requests

1. Provide a short description of the changes and screenshots of any visual changes.
2. Ensure that all status checks are passing.
3. You may merge the Pull Request in once you have the sign-off of one other developer, or if you do not have permission to do that, you may request the reviewer to merge it for you.
4. Once your PR is merged, changes should be reflected both in the Public Catalog and in New Relic One I/O within `4 hours`

### Contributor license agreement

Keep in mind that when you submit your Pull Request, you'll need to sign the CLA via the click-through using CLA-Assistant. If you'd like to execute our corporate CLA, or if you have any questions, please drop us an email at opensource@newrelic.com.

For more information about CLAs, please check out Alex Russell's excellent post,
["Why Do I Need to Sign This?"](https://infrequently.org/2008/06/why-do-i-need-to-sign-this/).

## Quickstart definitions

### Quickstarts
Quickstarts are higher level "bundles" of dashboards and alerts (components), plus the instrumentation required to power them. Some examples of things that should be quickstarts are Ruby on Rails, the LAMP stack, or Wordpress.

> They are defined under the `quickstarts/` directory and can be nested under organizational directories such as `aws`, `logging`, or `kuberneters`. Each quickstart has a `config.yml` file that defines metadata, components, install plans, and data sources.

Author: ```suggestion
They are defined under the `quickstarts/` directory and can be nested under organizational directories such as `aws`, `logging`, or `kubernetes`. Each quickstart has a `config.yml` file that defines metadata, components, install plans, and data sources.
```

- tracing
- windows

### Dashboards
Dashboards are defined under the `dashboards/` directory and CANNOT be nested, a dashboard named `example-dashboard` would live under `dashboards/example-dashboard`. They're defined using a JSON file and a set of screenshots. The dashboard JSON file is generated by exporting a dashboard from New Relic as JSON and removing/updating a few fields.

To convert an exported dashboard to the correct schema:
1. Remove the `permissions` field
2. Set every instance of `accountId` to `0`
3. Set every instance of `linkedEntityGuids` to `null`

> We provide a handy script to do this for you:

Author: Formatting is messed up here, probably needs a new line.

```json
{
  "name": "Dashboard Name",
  "description": null,
  "permissions": "PUBLIC_READ_WRITE"
}
```

#### Dashboard name uniqueness

- A dashboard's name must be unique. After providing a name in the `dashboard.json` file, you can check if your dashboard's name already exists by running `yarn run check_dashboard_name_uniqueness` from the `utils` directory.
- this [script](https://github.com/newrelic/newrelic-quickstarts/blob/main/utils/check_dashboard_name_uniqueness.js) will check and notify you of duplicate dashboard names in the repository.
- As a best practice you should run this script when creating a new dashboard.

#### Dashboard screenshots

- Dashboard images are `optional` but highly recommended to preview the visual functionality of a dashboard.
- File name should be `dashboard_name01`, `dashboard_name02`, etc
- Dashboards images should be stored in that dashboard's directory with its JSON file. ex: `dashboards/dashboard_name/dashboard_name01.png`.
- Must be in `.png`, `.jpg`, or `.jpeg` format
- Each image file must be less than `4MB` in size
- There should be no more than `12` dashboard or images per quickstart

> - For best results use aspect ratio: 3:2

Reviewer: Is this accurate for the current UI?

Author: That's a good question! Especially the aspect ratio portion. I'll have to go check the IO site
